components/Button/button.html

```diff
@@ -0,0 +1,208 @@
+          </div>
+          <div class="code-container">
+            <code>
+                <iframe
+                    src="https://carbon.now.sh/embed/3rpW2rZI5z6XuI6AlWvN"
+                    style="border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+                </code>
+          </div>
+        </div>
+        <div class="comp-wrapper">
+          <h3>Icon Button</h3>
+          <div class="content-container">
+            <button class="icon-btn">
+              <i class="fa-solid fa-cart-shopping-fast font-8"></i>
+            </button>
+            <button class="icon-btn">
+              <i class="fa-solid fa-envelope font-8"></i>
+            </button>
+          </div>
+          <div class="code-container">
+            <code><iframe
+                src="https://carbon.now.sh/embed/YRcJ6tBup0wcxhyWZti0"
+                style="border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+              </iframe> </code>
+          </div>
+        </div>
+        <div class="comp-wrapper">
+          <h3>Floating Action Button</h3>
+          <p class="text-medium">Click on Show to see Floating Action Button</p>
+          <div class="content-container">
+            <button class="btn" id="toggle-btn">Show</button>
+            <button class="btn float-btn" id="float-btn">
+                <span><i class="fa-regular fa-cart-plus"></i></span>
+                Add to cart
+            </button>
+        </div> 
+          <div class="code-container">
+            <code><iframe
+                src="https://carbon.now.sh/embed/Fm5LqQYft0rJkFLoHZmL"
+                style=" border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+              </iframe> </code>
+          
+        </div>
+      </main>
+      <footer class="footer">
+        <div class="footer-item footer-copy">Copyright © 2022 WINSOME-UI.</div>
+        <div class="footer-item">
+          <div class="footer-icons">
+            <a
+              href="https://github.com/AnjaliDeshwani"
+              target="_blank"
+              class="footer-link"
+            >
+              <i class="fa-brands fa-github"></i>
+            </a>
+          </div>
+
+          <div class="footer-icons">
+            <a
+              href="https://twitter.com/anjali_deshwani"
+              target="_blank"
+              class="footer-link"
+            >
+              <i class="fa-brands fa-twitter"></i>
+            </a>
+          </div>
+          <div class="footer-icons">
+            <a
+              href="https://www.linkedin.com/in/anjalideshwani/"
+              target="_blank"
+              class="footer-link"
+            >
+              <i class="fa-brands fa-linkedin-in"></i>
+            </a>
+          </div>
+        </div>
+      </footer>
+    </div>
+    <script>
```

Comment:
you could add this js in a separate js file, same as alert

Reply:
Yes corrected it now.

For primary color I also realized the same, so I have taken the darker shade and updated it in the dev branch it's not reflecting in this branch. Check it here-https://winsome-ui.netlify.app/

Reply:
yes, it looks good now